libs/facilities/frontend/view/src/lib/components/browse/browse.page.spec.ts

```diff
@@ -9,6 +10,12 @@ describe('BrowseComponent', () => {
 	beforeEach(async () => {
 		await TestBed.configureTestingModule({
 			imports: [XdBrowsePage],
+			providers: [
+				{
+					provide: ActivatedRoute,
+					useValue: {},
```

Comment:
Pls ensure the forwarding would be correct 
1. Provide a test route
2. Mock the facilities (provide them)
3. Perform button .click 
4. Check if the click works correctly

Moreover check if the data is displayed correctly  f.e. with .textConent.

Reply:
I implemented a very basic test to see if you are rerouted to the detail page if you click a button